mindsdb/api/http/start.py

```diff
@@ -75,11 +59,23 @@ def handle_exception(e):
 
     print(f"Start on {host}:{port}")
 
-    options = {
-        'bind': f'{host}:{port}',
-        'workers': min(max(multiprocessing.cpu_count(), 2), 3)
-    }
-    StandaloneApplication(app, options).run()
+    server = os.environ.get('MINDSDB_DEFAULT_SERVER', 'waitress')
```

Comment:
Seems fine, though I wonder if there's even need to keep `gunicorn` at all, thought I guess it might be somewhat superior to `waitress` in production.